INFO     root:conftest.py:226 === Starting test_negative_login_step2_invalid_token Test (Negative) ===
INFO     root:conftest.py:227 API Response: 401 | 
INFO     root:conftest.py:237 Test 'test_negative_login_step2_invalid_token' (Negative) SUCCESS.
INFO     root:conftest.py:238 === Ending test_negative_login_step2_invalid_token Test (Negative) ===